csrc/scheduler/matmul.cpp

```diff
@@ -906,11 +954,26 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
     splitk_sum = mma_result;
     mma_result = splitk_sum->rFactor({-4, -1});
```

Comment:
Is it possible to keep the `split(-4)` here but move the definition of `splitk_sum` and `rFactor` call inside `scheduleSplitKSum`? That is, we schedule smem epilogue and apply mma swizzle to `mma_result` first, then we do rFactor. This is just about code readability and make the code more modular, because as a code reader, I would not need to worry about split-k much when reading the code here, and I can only start thinking about split-k when reading `scheduleSplitKSum`.

Reply:
That is a good idea. We need to define `smem_epilogue` as `mma_result->cacheAfter()` and since `mma_result` is the intermediate tensor in the split-k rfactor, this means we can't define `smem_epilogue` until we've done the split-K rfactor. However, I think we could work around that and hide both `smem_epilogue` and `splitk_sum` in a new `scheduleEpilogue` function. Since this PR is functioning, I will merge as-is (pending CI) and begin a new PR to do that cleanup.